src/pc/lua/smlua_functions.c

```diff
@@ -946,6 +946,69 @@ int smlua_func_collision_find_surface_on_ray(lua_State* L) {
     return 1;
 }
 
+  ////////////////
+ // graph node //
+////////////////
+
+typedef struct { s16 type; u16 lot; } GraphNodeLot;
+static GraphNodeLot graphNodeLots[23] = {
```

Comment:
I recommend removing the manual count of graph node lots here, the compiler can automatically count this for you, and that would be useful for letting people add stuff to this list without accidentally forgetting to change this value